monai/networks/nets/hovernet.py

```diff
@@ -331,11 +365,12 @@ def __init__(
         )
 
         self.output_features.add_module(f"decoderblock{_i + 1}", _seq_block)
+        self.same_padding = same_padding
```

Comment:
Maybe we can remove this line? There seems no place has used it.

Reply:
Thanks! @KumoLiu !

Reply:
It's what enables this possibility (from _init_ docstring):

> decoder_padding: whether to do padding on convolution layers in the decoders. In the referred repository,
>             the architecture is changed to do padding on convolution layers in order to get the same output size
>             as the input. This changed version is used on CoNIC challenge.

Having said that, I don't think it's been tested and suspect that enabling it might break a few things, so perhaps it would be better to remove it for now.

Reply:
Hi @JHancox , I can add the unit tests and actually @KumoLiu has validated this option locally when she is trying to reproduce the performance on CoNIC dataset. What do you think?

Reply:
Hi @JHancox , thanks for posting this question, I double checked the network of conic version, and it only supports to produce same input size on "fast" mode. Therefore, I did some changes in the new commit and supplement some unit tests. Could you please help to recheck it? Thanks!